lib/percy/capybara/client.rb

```diff
@@ -42,8 +43,12 @@ def disable!
         @enabled = false
       end
 
+      def custom_loader?
```

Comment:
Drop this method to simplify things, just use `if custom_loader` as the check below and `expect(Percy::Capybara.capybara_client.custom_loader).to be` or `.not_to be` in the tests.